addon/build-waiter.ts

```diff
@@ -134,8 +137,19 @@ class NoopTestWaiter implements TestWaiter {
  * }
  */
 export default function buildWaiter(name: string): TestWaiter {
```

Comment:
Can you add a warning when using the same name twice?

```ts
const WAITER_NAMES = DEBUG ? new Set() : undefined;
export default function buildWaiter(name: string): TestWaiter {
  if (DEBUG) {
    warn('dont use the same name twice dummy', !WAITER_NAMES.has(name), { id: '@ember/test-waiters.duplicate-waiter-name' });
    WAITER_NAMES.add(name);
  }

  // ...snip...
}
```

Reply:
Yep. I specifically like the usage of 'dummy' in there.

Reply:
Added. Good suggestion.